.travis.yml

```
@@ -10,33 +10,33 @@ matrix:
   - python: "2.7_with_system_site_packages"
     env: TOXENV=py27-test
   - python: 2.7
-    env: TOXENV=py27-test,py27-bottle,py27-celery,py27-requests,py27-wsgi,py27-flask,py27-django18-migrate1-django1,py27-django19-migrate1-django1,py27-django110-migrate1-django1,py27-django111-migrate1-django1,py27-tornado
+    env: TOXFACTOR=py27
 
   - python: 3.3
     install:
     - travis_retry pip install coveralls 'virtualenv<16'
-    - pip install 'tox<3' 'pluggy<0.6.0' webtest
+    - pip install 'tox<3' 'pluggy<0.6.0' tox-factor webtest
     script:
-      - TOXENV=py33-test,py33-requests,py33-wsgi,py33-flask,py33-tornado tox
+      - TOXFACTOR=py33
```

This is the full test script, so it isn't running tests anymore because `tox` is missing

I think the line above can be `env` rather than `script` to match the rest, but GitHub won't let me leave a suggestion on that line :s

```suggestion
      - TOXFACTOR=py33 tox
```

---

Ah! nice catch.